src/assets/audio.rs

```diff
@@ -12,6 +12,25 @@ pub enum BGMusicType {
     Main,
 }
 
+#[derive(Deserialize, Debug, Hash, PartialEq, Eq, Clone, Display)]
```

Comment:
Move this  and `PlaySoundEffectEvent` to `thetawave_interface::audio`. Now there is a clear way for other plugins to play sound and allow this 1 module/compilation unit to deal with all audio assets, platform-specific audio libs/crap, etc.